essential-guide/filtering.md

```diff
@@ -0,0 +1,76 @@
+# Filtering
+
+Mews POS API supports filtering using the JSON:API reserved `filter` query parameter. The `filter` query parameter can be used to filter resources by attributes.
+
+Please note that support for filters is endpoint-specific. Visit the documentation of an endpoint to get more details about the supported filters.
+
+It is recommended to always URI-encode the values of the filter query parameters as filters can contain unsafe URL characters.
+
+### Supported filters
+
+| Property | Type | Contract | Description | Raw example | Encoded example |
```

Comment:
From experience with GitBook, I would recommend to redesign this table to have less columns. GitBook squeezes its content into quite a narrow space on the page and all table columns are equal-spaced. What this means is that deep tables work better than wide tables. Is it possible for example to put the Raw and Encoded examples in the Description column, using <br> to put them on new lines?

Reply:
@MikeAdamsMews I will give it a try.